secretmanager/snippets/regional_samples/delete_regional_secret.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google LLC
```

Comment:
The copyright year should reflect when the file was first introduced, so it should be 2024.

The years don't need to update when you edit the file, but should not be earlier than the first time the file is introduced.

```suggestion
# Copyright 2014 Google LLC
```